Vokoder.podspec

```diff
@@ -1,15 +1,16 @@
 Pod::Spec.new do |s|
   s.name             = "Vokoder"
-  s.version          = "4.1.1"
+  s.version          = "5.0.0"
```

Comment:
I would argue for 4.2.0
This isn't a breaking change for the existing platforms, yes?